### PR TITLE
tools/scylla-nodetool: implement the resetlocalschema command

### LINKED ISSUE #18468: nodetool does not support resetlocalschema
## Packages

Scylla version: `5.5.0~dev-20240429.af0c0ee8af48` with build-id `134767309d7a40060594dbbac80116e1828dbeae`

Kernel Version: `5.15.0-1061-azure`

## Issue description

- [X] This issue is a regression.
- [ ] It is unknown if this issue is a regression.

Trying to run `resetlocalschema` nemesis fails with error:
```
Command: '/usr/bin/nodetool  resetlocalschema '
Exit code: 100
Stdout:
Stderr:
error: unrecognized operation argument: expected one of (["checkAndRepairCdcStreams", "cleanup", "clearsnapshot", "compact", "compactionhistory", "compactionstats", "decommission", "describecluster", "describering", "disableautocompaction", "disablebackup", "disablebinary", "disablegossip", "drain", "enableautocompaction", "enablebackup", "enablebinary", "enablegossip", "flush", "getendpoints", "getlogginglevels", "getsstables", "gettraceprobability", "gossipinfo", "help", "info", "listsnapshots", "move", "netstats", "proxyhistograms", "rebuild", "refresh", "removenode", "repair", "ring", "scrub", "setlogginglevel", "settraceprobability", "snapshot", "sstableinfo", "status", "statusbackup", "statusbinary", "statusgossip", "stop", "tablehistograms", "cfhistograms", "tablestats", "cfstats", "toppartitions", "upgradesstables", "version", "viewbuildstatus"]), got resetlocalschema
```

## Impact

unable to run resetlocalschema

## How frequently does it reproduce?

Unknown. Possibly always.

## Installation details

Cluster size: 3 nodes (Standard_L8s_v3)

Scylla Nodes used in this run:
- pr-provision-test-speed-up-db-node-433e7280-eastus-3 (172.210.58.119 | 10.0.0.7) (shards: 7)
- pr-provision-test-speed-up-db-node-433e7280-eastus-2 (172.210.58.113 | 10.0.0.6) (shards: 7)
- pr-provision-test-speed-up-db-node-433e7280-eastus-1 (172.210.58.106 | 10.0.0.5) (shards: 7)

OS / Image: `/subscriptions/6c268694-47ab-43ab-b306-3c5514bc4112/resourceGroups/SCYLLA-IMAGES/providers/Microsoft.Compute/images/scylla-5.5.0-dev-x86_64-2024-04-30T02-54-24` (azure: undefined_region)

Test: `provision-test`
Test id: `433e7280-4092-47e8-88d4-2ee2a2c9ceff`
Test name: `scylla-staging/lukasz/provision-test`
Test config file(s):

- [PR-provision-test.yaml](https://github.com/scylladb/scylla-cluster-tests/blob/04945bfb8bc350ee82999b0bebcf092a1c3c0c8f/test-cases/PR-provision-test.yaml)

<details>
<summary>
Logs and commands
</summary>

- Restore Monitor Stack command: `$ hydra investigate show-monitor 433e7280-4092-47e8-88d4-2ee2a2c9ceff`
- Restore monitor on AWS instance using [Jenkins job](https://jenkins.scylladb.com/view/QA/job/QA-tools/job/hydra-show-monitor/parambuild/?test_id=433e7280-4092-47e8-88d4-2ee2a2c9ceff)
- Show all stored logs command: `$ hydra investigate show-logs 433e7280-4092-47e8-88d4-2ee2a2c9ceff`

## Logs:
- **db-cluster-433e7280.tar.gz** - [https://cloudius-jenkins-test.s3.amazonaws.com/433e7280-4092-47e8-88d4-2ee2a2c9ceff/20240430_071311/db-cluster-433e7280.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/433e7280-4092-47e8-88d4-2ee2a2c9ceff/20240430_071311/db-cluster-433e7280.tar.gz)
- **sct-runner-events-433e7280.tar.gz** - [https://cloudius-jenkins-test.s3.amazonaws.com/433e7280-4092-47e8-88d4-2ee2a2c9ceff/20240430_071311/sct-runner-events-433e7280.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/433e7280-4092-47e8-88d4-2ee2a2c9ceff/20240430_071311/sct-runner-events-433e7280.tar.gz)
- **sct-433e7280.log.tar.gz** - [https://cloudius-jenkins-test.s3.amazonaws.com/433e7280-4092-47e8-88d4-2ee2a2c9ceff/20240430_071311/sct-433e7280.log.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/433e7280-4092-47e8-88d4-2ee2a2c9ceff/20240430_071311/sct-433e7280.log.tar.gz)
- **loader-set-433e7280.tar.gz** - [https://cloudius-jenkins-test.s3.amazonaws.com/433e7280-4092-47e8-88d4-2ee2a2c9ceff/20240430_071311/loader-set-433e7280.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/433e7280-4092-47e8-88d4-2ee2a2c9ceff/20240430_071311/loader-set-433e7280.tar.gz)
- **monitor-set-433e7280.tar.gz** - [https://cloudius-jenkins-test.s3.amazonaws.com/433e7280-4092-47e8-88d4-2ee2a2c9ceff/20240430_071311/monitor-set-433e7280.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/433e7280-4092-47e8-88d4-2ee2a2c9ceff/20240430_071311/monitor-set-433e7280.tar.gz)
- **parallel-timelines-report-433e7280.tar.gz** - [https://cloudius-jenkins-test.s3.amazonaws.com/433e7280-4092-47e8-88d4-2ee2a2c9ceff/20240430_071311/parallel-timelines-report-433e7280.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/433e7280-4092-47e8-88d4-2ee2a2c9ceff/20240430_071311/parallel-timelines-report-433e7280.tar.gz)

[Jenkins job URL](https://jenkins.scylladb.com/job/scylla-staging/job/lukasz/job/provision-test/14/)
[Argus](https://argus.scylladb.com/test/fb89180f-241e-4341-9c64-bf57d9cbb0c3/runs?additionalRuns[]=433e7280-4092-47e8-88d4-2ee2a2c9ceff)
</details>

### PR DESCRIPTION
**Pull request by tchaikov**
Fixes #18468
Signed-off-by: Kefu Chai <kefu.chai@scylladb.com>

- [x] nodetool is a new feature in 5.5, no need to backport

### CODE REVIEW

File: tools/scylla-nodetool.cc
Comment (mykaul) [inline tools/scylla-nodetool.cc:3354]: Is that page supposed to be added with this PR? Otherwise, right now, it's 404.

Comment (denesb) [reply tools/scylla-nodetool.cc:3354]: We have this page, but only in master, not yet in stable: https://opensource.docs.scylladb.com/master/operating-scylla/nodetool-commands/resetlocalschema.html
See https://github.com/scylladb/scylladb/issues/18276.

**Review by denesb — APPROVED** [review]

File: tools/scylla-nodetool.cc
Comment (soyacz) [reply tools/scylla-nodetool.cc:3354]: works on 'master': https://opensource.docs.scylladb.com/master/operating-scylla/nodetool-commands/resetlocalschema.html
I think it was added recently - should be backported

Comment (tchaikov) [reply tools/scylla-nodetool.cc:3354]: @mykaul that page was added in dfa88ccc282477d7de14861a6278590ec378e055. guess we have not updated the "stable" build of the webpage. but presumably, we will once 5.5 is out.

**Comment by mykaul** [comment]
I wonder how this command works with schema over Raft btw.

**Comment by nyh** [comment]
It's getting harder and harder to understand all the strange lines in our CI output, but it doesn't seem like the build ran at all. Didn't even start. Anyone know why? Maybe one hour (!?) is just too short a time for the CI to even start? :-(

@tchaikov please try to get the CI to finish successfully before calling in the maintainers to merge it - @denesb has already approved it, so I guess he too was waiting for CI, and a good time to remind us will be after the CI finished.

**Comment by tchaikov** [comment]
@nyh i am sorry because i jumped the gun after seeing the "all green" CI, as you pointed out it's been one hour..

**Comment by nyh** [comment]
> @nyh i am sorry because i jumped the gun after seeing the "all green" CI, as you pointed out it's been one hour..

Yes, I complained about this several times, and @benipeled even created an issue https://github.com/scylladb/scylla-pkg/issues/3921, but it wasn't solved yet :-(

**Comment by mykaul** [comment]
> I wonder how this command works with schema over Raft btw.

@gleb-cloudius ?

**Comment by gleb-cloudius** [comment]
> > I wonder how this command works with schema over Raft btw.
> 
> @gleb-cloudius ?

The code here only calls already existing REST API from scylla-nodetool. The command is useless with raft, but should not hurt either.

**Pull request closed without merging**
